src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -220,6 +234,16 @@ def _alloc_from_dict(self, config_dict):
             self.config_path = config_dict[ConfigKeys.CONFIG_DIRECTORY]
         config_dict[ConfigKeys.CONFIG_DIRECTORY] = self.config_path
 
+        self.num_cpu_from_config = config_dict.get(ConfigKeys.NUM_CPU)
```

Comment:
Certain this key is found? Don't want to get with default?
`config_dict.get(ConfigKeys.NUM_CPU, 0)`

Reply:
that's a good point - the way it is designed, the default in this context is actually `None` - I can make it more explicit by saying `config_dict.get(ConfigKeys.NUM_CPU, None)`

that way (by using `None` rather than zero) we signal that res config did not find num cpu in user config.